packages/kbn-storybook/webpack.config.ts

```diff
@@ -70,6 +71,7 @@ export default function ({ config: storybookConfig }: { config: Configuration })
         },
       ],
     },
+    plugins: [new IgnoreNotFoundExportPlugin()],
```

Comment:
I feel like we should just be able to set https://webpack.js.org/configuration/stats/#statswarningsfilter here rather than using a custom plugin. We also do something similar in the optimizer, but we log stats ourselves so it's a little different, might make sense to share the regexp though https://github.com/elastic/kibana/blob/a7b5d3533eb96987b4e52fc7d952c1a96c9f2832/packages/kbn-optimizer/src/worker/webpack_helpers.ts#L38-L56

Reply:
We do have a warnings filter already: https://github.com/elastic/kibana/blob/master/packages/kbn-storybook/webpack.config.ts#L22, but I'm not sure how it affects what's output in the browser console.

Reply:
Ah, I see